src/test/java/com/github/dockerjava/core/command/InspectExecCmdImplTest.java

```diff
@@ -112,8 +119,19 @@ public void inspectExecNetworkSettings() throws IOException {
         assertThat(exec.getId(), not(isEmptyString()));
 
         InspectExecResponse inspectExecResponse = dockerClient.inspectExecCmd(exec.getId()).exec();
-        assertThat(inspectExecResponse.getExitCode(), is(0));
-
-        assertNotNull(inspectExecResponse.getContainer().getNetworkSettings().getNetworks().get("bridge"));
+        assertThat(inspectExecResponse.getExitCode(), is(nullValue()));
+        assertThat(inspectExecResponse.isOpenStdin(), is(false));
+        assertThat(inspectExecResponse.isOpenStdout(), is(true));
+        assertThat(inspectExecResponse.isRunning(), is(false));
+        assertThat(inspectExecResponse.getCanRemove(), is(false));
+        assertThat(inspectExecResponse.getContainerID(), is(container.getId()));
```

Comment:
Test differs between API versions at other places. Shouldn't that also count for ContainerID which is available only >=1.22?